examples/chem-sync-local-flask/manifest.yaml

```diff
@@ -3,8 +3,6 @@ manifestVersion: 1
 info:
   name: Sample Sync App
   version: 0.1.0
-settings:
-  lifecycleManagement: MANUAL
```

Comment:
Removed because it's incompatible with global apps and the manifest upload is rejected.